### ISSUE TITLE
Make openapi doc human readable

### ISSUE THREAD

**Issue by apomalyn** [ApplETS/whats-new-api#9]
As a developer, I would like to read easily the openapi documentation.

# Proposition

Using GitHub pages, and [this](https://github.com/apomalyn/etshome/tree/main/docs) you can make the YAML file pretty :wink:

**Comment by MysticFragilist** [comment]
Good idea! Right now when you launch the app in debug, you can see the documentation popping up automatically. Though integrating it through a github pages would be really cool!
![image](https://user-images.githubusercontent.com/25663435/162656177-6b60ecff-4b0a-43d8-aaf6-a103137ec667.png)

Also, I think your link is not working, maybe the repo is private? 😄

**Comment by apomalyn** [comment]
Oops 😅 sorry ahah [this one](https://github.com/apomalyn/pictural/tree/main/docs) should be good 😄 and the GitHub pages associated with it is [here](https://apomalyn.github.io/pictural/#get-/user) the UI can be changed of course to look like Swagger or OpenApi as wish of course

**Comment by MysticFragilist** [comment]
This should be handled with the deployment. It won't be on GitHub pages but will be accessible through the API. Will close when it's deployed and tested :)
